web/source/osk/visualKeyboard.ts

```diff
@@ -1313,19 +1313,19 @@ namespace com.keyman.osk {
       // Find minimum distance from any key
       var k, k0=0, dx, dxMax=24, dxMin=100000, x1, x2;
       for(k = 0; k < t.childNodes.length; k++) {
-        let childNode = t.childNodes[k] as HTMLElement; // gets the .kmw-key-square containing a key
-        if(childNode.firstChild) {
-          childNode = childNode.firstChild as HTMLElement; // gets the actual key element.
-        }
+        let keySquare = t.childNodes[k] as HTMLElement; // gets the .kmw-key-square containing a key
+        // Find the actual key element.
+        let childNode = keySquare.firstChild ? keySquare.firstChild as HTMLElement: keySquare;
+
```

Comment:
Compare against #3858.

At the time, I failed to realize the change's effect on the line below...